src/tests/functional/plugin/cpu/test_utils/fusing_test_utils.hpp

```diff
@@ -10,8 +10,15 @@
 
 namespace CPUTestUtils {
 
+struct postNodeConfig {
+    const std::shared_ptr<ngraph::Node> fuserNode;
```

Comment:
[Clarification]
fuserNode is the node we fuse into.
This info is necessary to generate correct shape which will be broadcasted in scope of fusing.
(Shape differs for MatMul/FC and the rest of the nodes)